packages/webpack-plugin/src/index.js

```diff
@@ -123,6 +123,25 @@ class LoadablePlugin {
             },
           )
         })
+
+        // update loadable-stats.json with integrity from webpack-subresource-integrity if available
+        compiler.hooks.done.tap(name, stats => {
+          const statsAssets = stats.toJson({ all: false, assets: true }).assets
```

Comment:
this line does the same as line 22 of the current file. 
There should be a way to run operation only once.